src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+interface Tag {
+  label: string;
+  color: string;
+}
+
+interface DataCardProps {
```

Comment:
We usually name the interfaces like this:

```suggestion
interface IDataCard {
```

and would be better to move this to the `interfaces.ts` file